vector/src/main/java/im/vector/app/features/home/room/list/SectionHeaderAdapter.kt

```diff
@@ -79,14 +80,17 @@ class SectionHeaderAdapter constructor(
         }
 
         fun bind(roomsSectionData: RoomsSectionData) {
-            binding.roomCategoryTitleView.text = roomsSectionData.name
-            val tintColor = ThemeUtils.getColor(binding.root.context, R.attr.vctr_content_secondary)
-            val expandedArrowDrawableRes = if (roomsSectionData.isExpanded) R.drawable.ic_expand_more else R.drawable.ic_expand_less
-            val expandedArrowDrawable = ContextCompat.getDrawable(binding.root.context, expandedArrowDrawableRes)?.also {
-                DrawableCompat.setTint(it, tintColor)
+            with(binding) {
```

Comment:
I am not a big fan of `with` (and `apply` for the same reason). It ends up with several `this` in the block and this is a source of bugs, and not good for the readability, especially on GitHub). We can discuss this later.

Reply:
On my side, I use a lot operators like `with`, `apply`, `run` or `let` when there are at least 2 operators applied to a variable. But I reverted back the usage here. I understand your point of view since it changes the scope, you must pay attention what you do inside.